frontend/src/components/LocationRealEstate.tsx

```diff
@@ -0,0 +1,76 @@
+
+const LocationRealEstate =()=>{
```

Comment:
file này nên đổi tên thành: 
B1: CardLocation
B2: Tạo 1 fake data json
B3: Code cái này theo dạng component rồi render ra
B4: Truyền vào cái CardLocation 1 cái prop có tên là isSmall để check (tức là card này có 2 dạng, 1 là to, 2 là nhỏ)
![image](https://github.com/dongpham2/homez/assets/80270016/80da1e6f-fe9a-4d83-98cd-86324760f56c)
![image](https://github.com/dongpham2/homez/assets/80270016/5e29fb6e-557f-4b76-80c0-56b90e52bb50)